docs/community/maintainer/pr-review.md

```diff
@@ -0,0 +1,16 @@
+# Pull Request Review Policy
+
+This document outlines the review policy for pull requests in the Trivy project.
+
+## Core Principles
+
+### 1. All Changes Through Pull Requests
+All changes to the `main` branch must be made through pull requests. Direct commits to `main` are not allowed.
+
+### 2. Required Approvals
+Every pull request requires approval from at least one CODEOWNER before merging.
+
+### 3. Merge Responsibility
+- **General Rule**: The pull request author should click the merge button after receiving required approvals
+- **Exception**: For urgent fixes (hotfixes), a CODEOWNER may merge the PR directly
+- **External Contributors**: Pull requests from external contributors should be merged by a CODEOWNER
```

Comment:
I think people expect this (contributing) section of the docs to be external facing, so this is probably the most relevant line in the doc. IMO in this doc we should document the approval process from external contributors eyes, and elsewhere we can document our internal processes.

Reply:
This is a document intended for maintainers, but are you suggesting that the maintainer directory should be moved somewhere outside the contributing section?

I'd prefer to keep Trivy-related information consolidated in one place as much as possible (if that information is not private), so either the documentation or the GitHub Wiki would be ideal. Actually, we used to maintain the documentation for maintainers in Microsoft Teams, but after it moved from the Wiki to Loop and went through several changes, it’s become hard to locate.